### ISSUE TITLE
entry descriptor loses protocol during parsing

### ISSUE THREAD

**Issue by selfisekai** [robertohuertasm/yarn-lock-parser#18]
This yarn.lock entry:
```
"minimatch@https://github.com/isaacs/minimatch.git#v10.0.1":
  version "10.0.1"
  resolved "https://github.com/isaacs/minimatch.git#0569cd3373408f9d701d3aab187b3f43a24a0db7"
  dependencies:
    brace-expansion "^2.0.1"
```

parses into this, losing the `https:` prefix:
```
Entry {
    name: "minimatch",
    version: "10.0.1",
    integrity: "",
    dependencies: [
        (
            "brace-expansion",
            "^2.0.1",
        ),
    ],
    descriptors: [
        (
            "minimatch",
            "//github.com/isaacs/minimatch.git#v10.0.1",
        ),
    ],
}
```

**Comment by selfisekai** [comment]
...intentional? This looks absolutely wrong to me.

https://github.com/robertohuertasm/yarn-lock-parser/blob/f589ed23b2dc9d7db30e35b6d44fa342ff198b55/src/lib.rs#L259-L260

**Comment by robertohuertasm** [comment]
Published [v0.8.0](https://crates.io/crates/yarn-lock-parser) with the changes.
